### PR TITLE
Add minutes for 07-22-2019 TSC meeting

### PR DESCRIPTION
**Pull request by michdolan**

### CODE REVIEW

File: docs/tsc/meetings/2019-07-22.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    - Michael: Thomas won't be there, but will provide statement to talk about
+      ACES 1.1 config.
+        - His work is mostly done, but not checked in anywhere yet, so we'll
+          likely be pointing people at his fork. Probably Ok since it's still
+          being tested.
+        - Michael: We're still trying to talk to Alex about what to do with
+          OCIO-Configs repo.
+        - Work will be included in Nuke 12 beta.
+        - Doug: Planning to show OCIO ACES implementation in comparison with
+          Thomas' config.
```
Comment (doug-walker) [inline docs/tsc/meetings/2019-07-22.md]: Doug: Will have a slide to talk about improved ACES Output Transform support we added.  Will have results vs. ACES 1.0.3 and Thomas' fork available on laptop for anyone interested.

File: docs/tsc/meetings/2019-07-22.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            - New shaper does fix some errors, but also introduces others. It
```
Comment (doug-walker) [inline docs/tsc/meetings/2019-07-22.md]: New shaper does improve performance on bright colors and is worth doing.  However performance in some areas does get worse, so it is moving the error around more than eliminating it.

File: docs/tsc/meetings/2019-07-22.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              moves the error around.
+            - ACES 1.2 is due out in the fall. Includes ACESclip.
```
Comment (doug-walker) [inline docs/tsc/meetings/2019-07-22.md]: - ACES 1.2 is due out in the fall. Includes new ACESclip and revised CLF spec.
 - ACES v2 (including Output Transform improvements, etc.) is still in planning phase.
 - Michael: The version of the ACES config following Thomas' 1.1 version will hopefully be able to use the new built-in OCIO transforms (next year sometime).

**Comment by michdolan** [comment]
Thanks for the corrections @doug-walker !

**Pull request merged**